src/main/target/common_pre.h

```diff
@@ -194,6 +194,7 @@
 #define USE_MSP_DISPLAYPORT
 #define USE_MSP_OVER_TELEMETRY
 #define USE_LED_STRIP
+#define USE_RX_RSSI_DBM
```

Comment:
This should probably go into the `#if ((FLASH_SIZE > 256) || (FEATURE_CUT_LEVEL < 1))` with the other CRSF extras.

Reply:
ok ill review .. and fix